packages/typescript-to-proptypes/test/typescript-to-proptypes.test.ts

```diff
@@ -94,7 +94,7 @@ describe('typescript-to-proptypes', () => {
       });
 
       if (fs.existsSync(outputPath)) {
-        expect(propTypes.replace(/\r?\n/g, '\n')).to.include(
+        expect(propTypes.replace(/\r?\n/g, '\n')).to.equal(
```

Comment:
Better failure message due to displaying a diff. `include` does not "include" a diff in the failure message.